bin/license_scout

```diff
@@ -39,3 +39,5 @@ collector.run
 report = collector.issue_report
 
 puts report
+
+exit 2 if !report.empty?
```

Comment:
Is there any verification we can do that this won't break any pipelines that are working or "working" now?